spring-integration-http/src/main/java/org/springframework/integration/http/outbound/AbstractHttpRequestExecutingMessageHandler.java

```diff
@@ -85,6 +82,8 @@
 	private static final List<HttpMethod> NO_BODY_HTTP_METHODS =
 			Arrays.asList(HttpMethod.GET, HttpMethod.HEAD, HttpMethod.TRACE);
 
+	DefaultUriBuilderFactory uriFactory = new DefaultUriBuilderFactory();
```

Comment:
Why package visibility?

Why not `UriBuilderFactory`?

Reply:
The `UriBuilderFactory` doesn't have a `setEncodingMode()`

Reply:
and package?